src/base/static/js/models/submission-model.js

```diff
@@ -0,0 +1,18 @@
+module.exports = Backbone.Model.extend({
+  sync: function(method, model, options) {
+    if (method === "update" && model.get("submitter")) {
+      // NOTE(jalmogo): If we are updating the place model and there
+      // is a submitter, we should omit the submitter from the payload so
+      // that the api knows to keep using the same reference. Otherwise,
+      // the api thinks we are posting a brand new submitter.
+      // Ideally, the api shouldn't allow for
+      // creating new submitters when creating/updating a Place.
+      // And we should be referencing the submitter by url
+      // on the Place, and storing a collection of Submitters,
+      // instead of hanging a submitter object off of the Place.
+      model.unset("submitter");
```

Comment:
addresses the "updating a Comment submitted by a logged in user" in jalMogo/mgmt#32